docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
+
+services:
+  db:
+    image: postgres
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: localdev
+    volumes:
+       - "./.postgres:/var/lib/postgresql/data"
+
+  adminer:
+    image: adminer
+    restart: always
+    ports:
+      - "8080:8080"
+
+  localstack:
+    image: localstack/localstack
+    ports:
+      - "127.0.0.1:4566:4566"            # LocalStack Gateway
+      - "127.0.0.1:4510-4559:4510-4559"  # external services port range
+    environment:
+      - DEBUG=1
+      - PERSISTENCE=1
+      - DOCKER_HOST=unix:///var/run/docker.sock
+      - SERVICES=s3,sqs
+    volumes:
+      - "./.localstack:/var/lib/localstack"
+      - "/var/run/docker.sock:/var/run/docker.sock"
```

Comment:
I don't use Windows myself, but are any of these referencing *nix-specific local paths  that would fail if someone tried to spin this up on Windows?

Reply:
yes, it seems to be working well on windows as well! (albeit confusing):
https://stackoverflow.com/a/42944785/38947

i followed the recommendation based off from the latest template they providing:

https://github.com/localstack/localstack/blob/master/docker-compose.yml

Note all volumes all relative, the only absolute one is the last line (but it should work in windows).